Pipfile

```diff
@@ -12,6 +12,7 @@ social-auth-app-django = "==4.0.0"
 social-auth-core = "==4.1.0"
 Django = "==2.2.20"
 PyJWT = "==2.0.1"
+mysqlclient = "*"
```

Comment:
Could this lead to some errors with certain versions?

Reply:
No, version is locked in the Pipfile.lock
```
        "mysqlclient": {
            "hashes": [
                "sha256:02c8826e6add9b20f4cb12dcf016485f7b1d6e30356a1204d05431867a1b3947",
                "sha256:2c8410f54492a3d2488a6a53e2d85b7e016751a1e7d116e7aea9c763f59f5e8c",
                "sha256:973235686f1b720536d417bf0a0d39b4ab3d5086b2b6ad5e6752393428c02b12",
                "sha256:b62d23c11c516cedb887377c8807628c1c65d57593b57853186a6ee18b0c6a5b",
                "sha256:e6279263d5a9feca3e0edbc2b2a52c057375bf301d47da2089c075ff76331d14"
            ],
            "index": "pypi",
            "version": "==2.1.0"
        },
```